mico-admin/src/app/app-detail/app-detail.component.ts

```diff
@@ -165,7 +166,7 @@ export class AppDetailComponent implements OnInit, OnDestroy {
     }
 
     /**
-     * abstract unsubscribe routine wiht null check
+     * generic unsubscribe routine wiht null check
```

Comment:
Maybe refactor unsubscribe methods into a utility module. (Also maybe change name to `safeUnsubscribe`)